docs/source/intro-clx-asset-classification.ipynb

```diff
@@ -0,0 +1,217 @@
+{
```

Comment:
<p>this would be more relatable if you subbed in "real" column names, like eventcode, taskcategory, logon_type... Since this example model doesn't have good accuracy anyway, you might as well make the sample data smaller but relatable. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>yeah, make sense. I just added the suggested changes.</p>